vendor-bin/phpunit/composer.json

```diff
@@ -0,0 +1,11 @@
+{
+    "require": {
+        "phpunit/phpunit": "^7.0"
```

Comment:
That's the part I like the least about this.

Reply:
Why don't you allow both versions? `^7.0|8.0`

Reply:
Because we commit the `composer.lock`, which means we will only ever use 7, and because we can't use 8-specific code anyway, since it would break the 7.1 pipeline.

Reply:
Well that's another thing. Why committing composer.lock? That's a pretty bad practice for libraries

Reply:
It's our policy: https://www.doctrine-project.org/policies.html

I don't like it any more than you do. But now that we are decoupled from libraries, maybe it makes sense to commit only lockfiles from tools, and not the global one?